### PR TITLE
STG-4: Profile Screen

### PR DESCRIPTION
**Pull request by ToniVinter**

### CODE REVIEW

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 1]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                const SizedBox(height: 48),
+                Container(
+                  decoration: const BoxDecoration(
+                    color: Colors.white,
+                    borderRadius: BorderRadius.all(Radius.circular(12)),
+                  ),
+                  child: Column(
+                    children: [
+                      ProfileTile(
+                        icon: Assets.notes,
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:148]: The color of the icon is a blueish (see figma)

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 2]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                Container(
+                  decoration: const BoxDecoration(
+                    color: Colors.white,
+                    borderRadius: BorderRadius.all(Radius.circular(12)),
+                  ),
+                  child: Column(
+                    children: [
+                      ProfileTile(
+                        icon: Assets.logout,
+                        title: 'Logout',
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:166]: logout does not have a counter (see figma). => remove '5'

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 3]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                Container(
+                  decoration: const BoxDecoration(
+                    color: Colors.white,
+                    borderRadius: BorderRadius.all(Radius.circular(12)),
+                  ),
+                  child: Column(
+                    children: [
+                      ProfileTile(
+                        icon: Assets.notes,
+                        title: 'Request a song',
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:149]: without counter

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 4]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                        icon: Assets.account,
+                        title: 'Account',
+                        totalNumber: '5',
+                        onTap: () {},
+                      ),
+                      _buildDivider(),
+                      ProfileTile(
+                        icon: Assets.account,
+                        title: 'Account',
+                        totalNumber: '5',
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:133]: without counter

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 5]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                        icon: Assets.notification,
+                        title: 'Notifications',
+                        totalNumber: '5',
+                        onTap: () {},
+                      ),
+                      _buildDivider(),
+                      ProfileTile(
+                        icon: Assets.account,
+                        title: 'Account',
+                        totalNumber: '5',
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:126]: without counter

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 6]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                  decoration: const BoxDecoration(
+                    color: Colors.white,
+                    borderRadius: BorderRadius.all(Radius.circular(12)),
+                  ),
+                  child: Column(
+                    children: [
+                      ProfileTile(
+                        icon: Assets.settings,
+                        title: 'App Settings',
+                        totalNumber: '5',
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:112]: without counter

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 7]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                        onTap: () {},
+                      ),
+                      _buildDivider(),
+                      ProfileTile(
+                        icon: Assets.notification,
+                        title: 'Notifications',
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:118]: without counter

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 8]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                              color: context.colorScheme.onSurfaceVariant),
+                        ),
+                      ],
+                    ),
+                    const Spacer(),
+                    const Column(
+                      children: <Widget>[
+                        CircleAvatar(
+                          backgroundImage:
+                              AssetImage('assets/images/profile1.png'),
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:63]: Image is larger than design, see figma

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 9]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                          radius: 40,
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/profile_screen.dart:64]: do not use radius, make it circle

File: lib/app/features/profile/presentation/widgets/profile_tile.dart [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+    return Container(
+      padding: const EdgeInsets.all(16),
+      child: Row(
+        children: [
+          ImageIcon(
+            AssetImage(icon),
+            size: 16,
+          ),
+          Padding(
+            padding: const EdgeInsets.only(left: 8.0),
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/widgets/profile_tile.dart:29]: Try using an 'int' literal.

File: lib/app/features/profile/presentation/widgets/profile_tile.dart [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+            child: Text(
+              title,
+              style: context.textTheme.bodyText1,
+              maxLines: 1,
+            ),
+          ),
+          const Spacer(),
+          Text(
+            totalNumber,
+            style: Theme.of(context).textTheme.subtitle1!.copyWith(
```
Comment (ToniVinter) [inline lib/app/features/profile/presentation/widgets/profile_tile.dart:39]: subtitle1 is deprecated

Comment (ToniVinter) [inline lib/app/features/profile/presentation/widgets/profile_tile.dart:39]: use context.textTheme

**Review by ToniVinter — COMMENTED** [review]
Please modify according to this review

File: lib/app/features/profile/presentation/profile_screen.dart (hunk 9 above)
Comment (BeniaminSzilagyi) [reply lib/app/features/profile/presentation/profile_screen.dart:64]: I don't understand the request

File: lib/app/features/profile/presentation/widgets/profile_tile.dart (hunk 10 above)
Comment (BeniaminSzilagyi) [reply lib/app/features/profile/presentation/widgets/profile_tile.dart:29]: I don't understand ?

File: lib/app/features/profile/presentation/profile_screen.dart [hunk 12]
```diff
@@ -17,18 +30,149 @@ class _ProfileScreentate extends State<ProfileScreen> {
...
+                      children: <Widget>[
+                        Text(
+                          'John Mayer',
+                          style: context.textTheme.headlineLarge,
+                        ),
+                        const SizedBox(height: 8),
+                        Text(
+                          '@johnmayer03',
+                          style: context.textTheme.bodyLarge!.copyWith(
+                              color: context.colorScheme.onSurfaceVariant),
```
[the remark this answers is not in the document]
Comment (BeniaminSzilagyi) [reply lib/app/features/profile/presentation/profile_screen.dart:54]: to add an extra comma or to delete?

**Pull request merged**